watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/supercritical_sludge_to_gas/supercritical_sludge_to_gas.py

```diff
@@ -168,7 +171,7 @@ def solve(blk, solver=None, tee=False, check_termination=True):
         solver = get_solver()
     results = solver.solve(blk, tee=tee)
     if check_termination:
-        assert_optimal_termination(results)
+        check_optimal_termination(results)
```

Comment:
Same thing here as noted above. Actually in this case, I think we can keep `assert_optimal_termination`.